src/libmongoc/tests/test-mongoc-sdam.c

```diff
@@ -250,6 +495,15 @@ test_all_spec_tests (TestSuite *suite)
    ASSERT (realpath (JSON_DIR "/server_discovery_and_monitoring/supplemental",
                      resolved));
    install_json_test_suite (suite, resolved, &test_sdam_cb);
+
+   /* Integration tests. */
+   ASSERT (realpath (JSON_DIR "/server_discovery_and_monitoring/integration",
+                     resolved));
+   install_json_test_suite_with_check (suite,
+                                       resolved,
+                                       &test_sdam_integration_cb,
+                                       test_framework_skip_if_no_crypto,
```

Comment:
Do these tests actually care about whether crypto is available, or was this copied from transactions (which require sessions)?